config/http/handler/handlers/redirect.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^3.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "Example handler to configure redirect patterns. Add to urn:solid-server:default:HttpHandler",
+      "@id": "urn:solid-server:default:RedirectHandler",
+      "@type": "RedirectingHttpHandler",
```

Comment:
Can I leave that up to you?

Reply:
Sure. But with the changes now removing the `RedirectAllHttpHandler` this branch should be rebased and target the v4.0.0 branch since this is a breaking change (assuming that change works, will look at the other issue below 😅 ).